network-api/networkapi/mozfest/migrations/0005_auto_20190730_2045.py

```diff
@@ -0,0 +1,33 @@
+# Generated by Django 2.2.3 on 2019-07-30 20:45
+
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('mozfest', '0004_mozfestprimarypage_use_wide_template'),
+    ]
+
+    operations = [
+        migrations.AddField(
+            model_name='mozfestprimarypage',
+            name='cta_button_destination',
+            field=models.CharField(blank=True, help_text='The URL for the page that the CTA button in the primary nav bar should redirect to', max_length=2048),
+        ),
+        migrations.AddField(
+            model_name='mozfestprimarypage',
+            name='cta_button_label',
+            field=models.CharField(blank=True, help_text='Label text for the CTA button in the primary nav bar', max_length=250),
+        ),
+        migrations.AlterField(
```

Comment:
I'm not sure why wagtail/django altered these fields but... this is an auto-generated migration, so I guess it feels it needs those alterations? =S